lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -1411,3 +1411,95 @@ func (p *ConnPool) TTL(table, key string) (int, *errors.Error) {
 	}
 	return time, nil
 }
+
+// CreateAggregateSubscriptionIndex is used to create and save secondary index
+/* CreateAggregateSubscriptionIndex take the following keys are input:
+1. index is the name of the index to be created
+2. key is for the index
+*/
+func (p *ConnPool) CreateAggregateSubscriptionIndex(index, aggregateID string, hostIP []string) error {
```

Comment:
Rename this function it shouldn't be subscription index as we discussed

Reply:
done